File: src/main/java/net/hectus/neobb/turn/default_game/other/OtherTurn.java
package net.hectus.neobb.turn.default_game.other;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.turn.Turn;
import org.bukkit.Location;

public abstract class OtherTurn<T> extends Turn<T> {
    public OtherTurn(NeoPlayer player) { super(null, null, player); }
    public OtherTurn(T data, Location location, NeoPlayer player) { super(data, location, player); }
}

File: src/main/kotlin/net/hectus/neobb/turn/default_game/other/OtherTurn.kt
package net.hectus.neobb.turn.default_game.other

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import net.hectus.neobb.turn.Turn

abstract class OtherTurn<T>(data: T?, cord: Cord?, player: NeoPlayer?) : Turn<T>(data, cord, player)